lib/renderer/extensions/event.ts

```diff
@@ -1,26 +1,22 @@
-'use strict'
-
 class Event {
-  constructor () {
-    this.listeners = []
-  }
+  private listeners: Function[] = []
 
-  addListener (callback) {
+  addListener (callback: Function) {
     this.listeners.push(callback)
   }
 
-  removeListener (callback) {
+  removeListener (callback: Function) {
     const index = this.listeners.indexOf(callback)
     if (index !== -1) {
       this.listeners.splice(index, 1)
     }
   }
 
-  emit (...args) {
+  emit (...args: any) {
     for (const listener of this.listeners) {
       listener(...args)
     }
   }
 }
 
-module.exports = Event
+export default Event
```

Comment:
Can we prefer named exports for things not in `*/api/*`.  E.g. this should be `export class Event`